src/class-wpml-elementor-translatable-nodes.php

```diff
@@ -59,9 +59,13 @@ public function get( $node_id, $element ) {
 							if ( $class instanceof \WPML_Elementor_Module_With_Items ) {
 								$node = $class;
 							} else {
-								$node = new $class();
+								if ( class_exists( $class ) ) {
```

Comment:
@GeorgeBotsev, we can merge this into `elseif`:

```php
} elseif ( class_exists( $class ) ) {
    $node = new $class();
}
```

Reply:
I noticed that we have the instantiation of the class at the other place too, but I did not get any error there when I tested.

Reply:
@GeorgeBotsev, this is a good example of duplicated code where we have to apply the same fix in 2 different places. What we should do here is to refactor a little bit so we just have to fix in 1 place.

My suggestion is to replace the `get_integration_classes` method by `get_integration_instances` which will return a filtered array (only object instances).

Reply:
Here's the idea @GeorgeBotsev https://github.com/OnTheGoSystems/wpml-page-builders-elementor/commit/438747060e9e54e3600279ca1aa3952a04c4f7ab.

It's on the branch https://github.com/OnTheGoSystems/wpml-page-builders-elementor/commits/wpmlcore-7433-pierre.

Reply:
Ok, I tested your suggestion and I applied the fix to check if the class exist, and I am implementing it in the code as a commit now.

Reply:
@GeorgeBotsev, I suggest to separate the refactoring and the actual fix in 2 separate commits:
1. Re-factor duplicated code 
2. Prevent fatal error if integration class does not exist

This is a good pratice because:
1. We can make sure first that the re-factoring is not breaking the existing tests (the refactoring commit should not include any change in the tests).
2. It's easier to see the actual fix in the review (we can display only the commit related to the fix).

Moreover, I think you can also add a small unit test for the new case that you fixed (just pass a class name that does not exist, `An_Unexisting_Class`). This is not critical, but a good exercise for you to understand how we can progressively increase the constraints on our code (we don't change our tests, we just add more).